src/dependencies/dss-flappers/FlapperInit.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.8.0;
+
+import { DssInstance } from "dss-test/MCD.sol";
+
+struct FlapperInstance {
```

Comment:
Any special reason why not to copy `FlappeInstance.sol` file as well?
I know there's not much there but the idea is that these files can be diffed automatically (maybe even by some tool in the future).

Reply:
It seemed too much overhead and I think this way was done for D3M. But not problem, I'll add it here.

Reply:
Done.